library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Rudimentary-spike-wave-complex <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Synonym: Pseudo petit mal discharge. Paroxysmal discharge that consists of generalized or nearly generalized high voltage 3 to 4/sec waves with poorly developed spike in the positive trough between the slow waves, occurring in drowsiness only. It is found only in infancy and early childhood when marked hypnagogic rhythmical theta activity is paramount in the drowsy state. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Slow-fused-transient <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [A posterior slow-wave preceded by a sharp-contoured potential that blends together with the ensuing slow wave, in children. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Needle-like-occipital-spikes-blind <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Spike discharges of a particularly fast and needle-like character develop over the occipital region in most congenitally blind children. Completely disappear during childhood or adolescence. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Subclinical-rhythmic-EEG-discharge-adults <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Subclinical rhythmic EEG discharge of adults (SERDA). A rhythmic pattern seen in the adult age group, mainly in the waking state or drowsiness. It consists of a mixture of frequencies, often predominant in the theta range. The onset may be fairly abrupt with widespread sharp rhythmical theta and occasionally with delta activity. As to the spatial distribution, a maximum of this discharge is usually found over the centroparietal region and especially over the vertex. It may resemble a seizure discharge but is not accompanied by any clinical signs or symptoms. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
```

Comment:
Should adopt a consistent cap rule for abbreviations such as SERDA

Reply:
"in the adult age group" -->  "in adults"

Reply:
cap rule will be all caps for abbreviations, fixed issue